server/routes/api.js

```diff
@@ -456,8 +460,6 @@ router.post('/messages', function(req, res, next) {
         var pdwMode = nconf.get('messages:pdwMode');
         var pushenable = nconf.get('pushover:pushenable');
         var pushkey = nconf.get('pushover:pushAPIKEY');
-        var pushgroup = nconf.get('pushover:pushgroup');
-        var pushSound = nconf.get('pushover:pushSound');
```

Comment:
For backwards compatibility it might be wise to retain this - maybe keep the config file vars, have a test for if the group/sound columns are non-empty in the DB row then use those, otherwise use the defaults specified in the config?

Reply:
It *seems* to not cause any issues leaving them out - i've copied my production database over to my test one running this version without issue. 
The only problem will be that pushover will need to be reconfigured on the alias level to work after upgrading. 

I thought about writing in a way for it to use the values set on the settings screen if there was nothing set at the alias level, but then i realized i have no idea how to do that ;)

Might be something for me to do over the weekend.....